admin/app/assets/javascripts/workarea/admin/modules/toggle.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
thought we have this behavior already, but again I'll defer to Curt

Reply:
yeah, I thought so too 😞

Reply:
I did as well. I'll triple check as part of #142.